From fecd2b53af43c4504a7d345a30d9f98954801167 Mon Sep 17 00:00:00 2001
From: Baptiste Assmann <bedis9@gmail.com>
Date: Fri, 14 Aug 2015 16:20:52 +0200
Subject: [PATCH 15/28] MINOR: init: server state loaded from file

With this patch, HAProxy reads the content of server state file and
update state of servers accordingly.
---
 src/haproxy.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/haproxy.c b/src/haproxy.c
index 681109d..465bb6a 100644
--- a/src/haproxy.c
+++ b/src/haproxy.c
@@ -800,6 +800,9 @@ void init(int argc, char **argv)
 		exit(2);
 	}
 
+	/* Apply server states */
+	apply_server_state();
+
 	global_listener_queue_task = task_new();
 	if (!global_listener_queue_task) {
 		Alert("Out of memory when initializing global task\n");
-- 
2.4.9

